src/lib.rs

```diff
@@ -1,14 +1,6 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
-}
+pub mod error;
+pub mod message;
+mod utils;
 
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn it_works() {
-        let result = add(2, 2);
-        assert_eq!(result, 4);
-    }
-}
+pub use message::{Message, MessageList, CesrGroup, CustomPayload, Group};
+pub use cesride::{Cigar, Siger, Signer, Dater, Verfer, Saider, Diger, Matter, Indexer, Prefixer, Seqner};
```

Comment:
Do we need to expose cesride primitives here or can we assume the Rust consumer will just use from both packages?

Reply:
we do not need anymore. I'll remove this line